test/end-to-end/pages/screeners/deathcertificatetranslation.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
+const deathCertTranslationEn = require('app/resources/en/translation/screeners/deathcertificatetranslation');
+const deathCertTranslationCy = require('app/resources/cy/translation/screeners/deathcertificatetranslation');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/deathcertificatetranslation');
-
-module.exports = async function(answer) {
+module.exports = async function(language ='en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
+    const deathCertTranslationContent = language === 'en' ? deathCertTranslationEn : deathCertTranslationCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/deathcertificatetranslation');
-    await I.waitForText(content.question);
-
+    if (language === 'en') {
+        await I.waitForText(deathCertTranslationContent.question);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages